janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementLogger.java

```diff
@@ -191,12 +211,24 @@ public void run() {
                         txStillOpen = true;
                     }
                 }
-                if (!txStillOpen) {
+                final JanusGraphManager jgm = JanusGraphManager.getInstance();
+                final boolean janusGraphManagerIsInBadState = null == jgm && action.equals(EVICT);
+                if (!txStillOpen && janusGraphManagerIsInBadState) {
```

Comment:
Looks like this branch still isn't getting hit (see [here](https://codecov.io/gh/JanusGraph/janusgraph/src/graph-cache-evictions/janusgraph-core/src/main/java/org/janusgraph/graphdb/database/management/ManagementLogger.java#L214...221)). It's not a big deal.